From: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Date: Mon, 19 Aug 2019 16:17:20 +0100
Subject: ubuntu_privacy_hide_buggy_controls

# Description: don't list non working camera/microphones items in 

# Description: don't list non working camera/microphones items in
# the privacy settings
# Upstream: https://gitlab.gnome.org/GNOME/gnome-control-center/issues/381
---
 panels/privacy/cc-privacy-panel.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/panels/privacy/cc-privacy-panel.c b/panels/privacy/cc-privacy-panel.c
index 6256745..4deba02 100644
--- a/panels/privacy/cc-privacy-panel.c
+++ b/panels/privacy/cc-privacy-panel.c
@@ -1641,8 +1641,13 @@ cc_privacy_panel_init (CcPrivacyPanel *self)
 
   add_screen_lock (self);
   add_location (self);
+  /* Hide buggy camera/microphone controls, 
+     https://gitlab.gnome.org/GNOME/gnome-control-center/issues/381
+     
   add_camera (self);
   add_microphone (self);
+  */
+
   add_usage_history (self);
   add_trash_temp (self);
   add_software (self);
